### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by ChrisRackauckas** [nlw0/SparseBlockJacobians.jl#2]

**Comment by nlw0** [comment]
Yes, tests, docs, benchmark, package... I don't even know if I like the API, and I'm opening this project early on to get some feedback and decide what to build in the first place.
